terraso_backend/apps/graphql/schema/memberships.py

```diff
@@ -100,6 +101,7 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
 
         if group.membership_type == Group.MEMBERSHIP_TYPE_CLOSED:
             membership.membership_status = Membership.PENDING
+            EmailNotification.SendMembershipRequest(user, group)
```

Comment:
Issue: function start with a lowercase character and should be snake cased: `send_membership_request `